constants/chapter.json

```diff
@@ -47,9 +47,43 @@
     "title": "Chapter 3",
     "subtitle": "Functions",
     "image": "/chapter-3.jpg",
-    "exercises": "",
+    "exercises": "Minimum, Recursion, Bean counting",
     "link": "/chapter3",
-    "details": ""
+    "details": "This chapter taught you how to write your own functions. Separating the tasks your program performs into different functions is helpful. You won’t have to repeat yourself as much, and functions can help organize a program by grouping code into pieces that do specific things.",
+    "devOscar": [
+      {
+        "id": 1,
+        "title": "Minimum",
+        "details": "Press the button below to display the youngest person."
+      },
+      {
+        "id": 2,
+        "title": "Recursion",
+        "details": "So I was thinking about recursion, then I found a flower and I came up with this:"
```

Comment:
For consistency, I would remove the trailing `:`

Reply:
refactored